src/components/v5/frame/ColonyHome/partials/ReputationChart/utils.ts

```diff
@@ -89,50 +90,69 @@ export const getTeamReputationChartData = (
   const reputationInOtherTeams = 100 - topTeamsTotalReputation;
```

Comment:
I still believe this line should be refactored into
```
const reputationInOtherTeams  = domainsWithoutRoot
    .slice(WIDGET_TEAM_LIMIT)
    .reduce(
      (reputation, team) => reputation + Number(team.reputation || 0),
      0,
    );
```
Because with this implementation I artificially considered there might be some reputation in the other domains if the `topTeamsTotalReputation` isn't `100%`

And then your call to adjust the values inside `getTeamReputationChartData` will not really have an effect. 
What do you think?

Reply:
Yes, let's do it, seems more proper